src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -274,3 +275,23 @@
 (defn get-balance-for-chain
   [data chain-id]
   (some #(when (= chain-id (:chain-id %)) %) (vals data)))
+
+(defn make-network-item
+  "This function generates props for quo/category component item"
+  [{:keys [network-name] :as _network}
+   {:keys [title color on-change networks state label-props] :as _options}]
+  (cond-> {:title        (or title (string/capitalize (name network-name)))
```

Comment:
I think during this month we have to refactor we should address the:
`(string/capitalize (name network-name))`
usages

Reply:
Yea, good shout @ulisesmac

Reply:
Yes, @ulisesmac - I already started to ideate on how we can improve it. Since this PR is just for UI implementation. I left it as it is. I will do some refactors along with this feature implementation. Additionally, this method `make-network-item` doesn't need to receive all the keys from the network. We will improve gradually. 👍